agent/api/container/container.go

```diff
@@ -401,8 +401,8 @@ func (c *Container) GetNextKnownStateProgression() apicontainerstatus.ContainerS
 	return c.GetKnownStatus() + 1
 }
 
-// IsInternal returns true if the container type is either `ContainerEmptyHostVolume`
-// or `ContainerCNIPause`. It returns false otherwise
+// IsInternal returns true if the container type is `ContainerEmptyHostVolume`,
```

Comment:
We don't use the `ContainerEmptyHostVolume` anymore, it can be removed from the comment.